flepimop/R_packages/inference/R/install_cli.R

```diff
@@ -0,0 +1,32 @@
+
+#' @title Install Inference Scripts
+#' 
+#' @description
+#' Installs the scripts for R-based FlepiMoP inference.
+#' 
+#' @param path The path to install the scripts to. Default is `usr/local/bin` (unix-like).
+#' 
+#' @param overwrite Whether to overwrite existing scripts. Default is `TRUE`. see [base::file.copy()].
+#' 
+#' @export
+install_cli <- function(
+  path = {
+    condapth <- Sys.getenv("CONDA_PREFIX")
+    if (condapth != "") { 
+      file.path(condapth, "bin")
+    } else {
+      stop("only support default path installation when conda is running.")
+    }
+  }
+) {
+  scriptfiles <- list.files(
+    system.file("scripts", package = utils::packageName()), pattern = "flepimop-.*", full.names = TRUE
+  )
+  from <- scriptfiles
+  to <- file.path(path, gsub("\\.R$", "", basename(scriptfiles)))
+  to_remove <- file.exists(to)
+  if (length(to_remove)) {
+    file.remove(to)
+  }
```

Comment:
Added this to get rid of a warning about the file existing on reinstall that looks like this:
```R
> library(inference); inference::install_cli()
[1] FALSE FALSE
Warning messages:
1: In file.symlink(from, to) :
  cannot symlink '/users/t/w/twillard/flepimop-env/lib/R/library/inference/scripts/flepimop-inference-main.R' to '/users/t/w/twillard/flepimop-env/bin/flepimop-inference-main', reason 'File exists'
2: In file.symlink(from, to) :
  cannot symlink '/users/t/w/twillard/flepimop-env/lib/R/library/inference/scripts/flepimop-inference-slot.R' to '/users/t/w/twillard/flepimop-env/bin/flepimop-inference-slot', reason 'File exists'
```
Are there side effects from doing this that I'm missing @pearsonca?

Reply:
this should be fine. if we don't change the names of the target files, we actually don't need to rewrite the symlinks when updating the file.

but i'm iffy enough on that if that it seems best to always blowaway and recreate the symlink

Reply:
Alternative is `suppressWarnings(file.symlink(from, to))` which feels heavy handed and leaves the door open to suppressing an actual warning though.